### DISCUSSION TITLE
Control, Modeling, Design, and Simulation of Modern HVAC Systems (Project 195)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#26] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project #195](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Control%2C%20Modeling%2C%20Design%2C%20and%20Simulation%20of%20Modern%20HVAC%20Systems)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by Suyash7j** [comment]
Hi, I am a student at Purdue University and looking forward to collaborating with fellow researchers on this project

**Reply by saad687** [reply]
Hi Suyash! I wish to work on this project as well. Shall we work together?

**Reply by rajeshmahadeva** [reply]
Hi Suyash! I am Rajesh Mahadeva, PhD Scholar, IIT Roorkee, I wish to work on this project as well. Shall we work together?

**Reply by NIKHIL2498-blip** [reply]
Hello, I'm a master's student at Coventry University and would like to work collaboratively and need some support on this project.

**Reply by robertogl** [reply]
Hello @NIKHIL2498-blip and all,

Great to know you will be working on this project. If you have any question, please write it here and we will do our best to answer as soon as possible!

**Reply by Armaan2227** [reply]
hey i did this project done ?

**Reply by robertogl** [reply]
Hi @Armaan2227,

If you completed the project, please fill out the submission form available in the project description page.
Let me also paste the link [here](https://www.mathworks.com/academia/student-challenge/mathworks-excellence-in-innovation-submission-form.html?tfa_1=Control,%20Modeling,%20Design,%20and%20Simulation%20of%20Modern%20HVAC%20Systems&tfa_2=195).

**Comment by amnforum** [comment]
Hi, I am a student at MSRIT and interested in working and collaborating on this project

**Reply by francis-web** [reply]
i am also interested

**Comment by thejah2001** [comment]
Hello!
This is Theja from India. I would like to work on this project and be a part of the team. Would love to collaborate.

**Comment by francis-web** [comment]
That'll be very great.
Welcome in the team and nice to have you with us.

+233 509890546

**Reply by thejah2001** [reply]
How do I get to know the team?

**Comment by ARNAV293** [comment]
Hello this is Arnav, from VIT VELLORE, Currently working on the project

**Reply by Armaan2227** [reply]
hi did u done this ?

**Comment by thejah2001** [comment]
Hello,

How long have you been working on this?

Regards,
Theja.

**Comment by francis-web** [comment]
Cam i take a look?
